src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,73 @@
+$color-light-blue: rgba(0, 172, 220, 1);
+
+.product-card {
+  padding: 32px 16px 16px;
+  margin-bottom: 50px;
+  width: 166px;
+  border: 1px solid rgba(229, 229, 229, 1);
+  border-radius: 5px;
+
+  &--bottom {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    width: 160px;
+    padding: 0 3px;
+  }
```

Comment:
Why do you need these paddings? Try to rework the width of your card over the mockup. There is a fixed width of 200. In that way you won't need to set up paddings like that